bedrock/mozorg/views.py

```diff
@@ -117,7 +117,7 @@ def home_view(request):
 
     if locale.startswith('en-'):
         template_name = 'mozorg/home/home-en.html'
-        ctx['page_content_cards'] = get_page_content_cards('home-en', 'en-US')
+        ctx['page_content_cards'] = get_page_content_cards('home-2020-oct', 'en-US')
```

Comment:
I'd drop the `oct` from the folder name since there's a reasonable chance we'll be updating the cards in November, but not changing the layout. (need to change in www-admin too). Not a blocker though.

Reply:
Revised!